mmdeploy/codebase/mmseg/models/segmentors/encoder_decoder.py

```diff
@@ -25,6 +25,14 @@ def encoder_decoder__predict(self, inputs, data_samples, **kwargs):
         batch_img_metas.append(data_sample.metainfo)
     x = self.extract_feat(inputs)
     seg_logit = self.decode_head.predict(x, batch_img_metas, self.test_cfg)
+
```

Comment:
why do we need marker here? Is your model partition policy same as RKNN which skips the last `ArgMax` op in ONNX.

Reply:
Yes. `ArgMax` not support in vacc backend

Reply:
Then you could remove newly added rewritings and add to existing rewriting https://github.com/open-mmlab/mmdeploy/blob/b5fef4873a085a0c843f7338593e948b17339876/mmdeploy/codebase/mmseg/models/segmentors/encoder_decoder.py#L32